### PR TITLE
Add Introduction to Web Neural Network API (WebNN)

### PR DESCRIPTION
**Pull request by ibelem**
@huningxin  PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,26 +116,27 @@
 
 ## Tutorials
 
-### MDN Docs
+## Hugging Face Transformers
```
Comment (huningxin) [inline README.md]: Just "Transformers.js"

File: README.md [hunk 2]
```diff
@@ -110,26 +116,27 @@
...
 
-- [WebNN](https://developer.mozilla.org/en-US/search?q=webnn) - To do
+- To do
 
-### WebNN API
+### Google JS ML Framework
```
Comment (huningxin) [inline README.md]: Do you mean TensorFlow Lite? If we don't have clear target at the current stage, we can remove it in this PR and add it later.

File: README.md [hunk 3]
```diff
@@ -110,26 +116,27 @@
...
+- To do
 
-### Based on Google JS ML Framework
+### MDN Docs
 
-- To do
+- [WebNN](https://developer.mozilla.org/en-US/search?q=webnn) - To do
 
-### Based on ONNX Runtime Web
+### Microsoft ONNX Runtime Web
```
Comment (huningxin) [inline README.md]: Just "ONNX Runtime Web"

File: README.md (hunk 1 above)
Comment (ibelem) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (ibelem) [reply README.md]: Removed

File: README.md (hunk 3 above)
Comment (ibelem) [reply README.md]: Fixed

**Review by huningxin — APPROVED** [review]
LGTM

**Comment by ibelem** [comment]
Thanks a lot for the review @huningxin !

**Pull request merged**